### DISCUSSION TITLE
邀請做加香黑客松 (Alpha) 既 Penalist

### DISCUSSION THREAD

**Discussion by elton-lau** [Pjaijai/Referalah#321] [平台需要穚 | Ideas]
話說好多 work break 緊既朋友會去做 side project 準備面試，我地想透過呢 d 機會幫 NGO 或者多倫多 SME 去解決吓問題。
目前我地搵到間公司同我地合作一齊試吓水溫，唔知有你無興趣做我地既 panelist 呢？
有需要歡迎你地隨時過黎搵我，我地可以再詳談

**Comment by Pjaijai** [comment]
Hi! 聽落幾有趣，想了解吓詳情，請射個email 去r1r69.referalah@gmail.com，謝謝🙏🏻
